prow/tide/gerrit.go

```diff
@@ -455,10 +455,12 @@ func (p *GerritProvider) GetPresubmits(identifier string, baseSHAGetter config.R
 
 func (p *GerritProvider) GetChangedFiles(org, repo string, number int) ([]string, error) {
 	// "CURRENT_FILES" lists all changed files from current revision, which is
-	// what we want, "CURRENT_REVISION" is required for "CURRENT_FILES"
+	// what we want, "CURRENT_REVISION" is required for "CURRENT_FILES". Also
+	// "LABELS" are required for figuring out whether a Gerrit label is required
+	// for submission or not.
 	// according to
 	// https://gerrit-review.googlesource.com/Documentation/rest-api-changes.html#list-changes.
-	change, err := p.gc.GetChange(org, strconv.Itoa(number), "CURRENT_FILES", "CURRENT_REVISION")
+	change, err := p.gc.GetChange(org, strconv.Itoa(number), "CURRENT_FILES", "CURRENT_REVISION", "LABELS")
```

Comment:
Why are `LABELS` needed here? They don't appear to be used: https://github.com/chaodaiG/test-infra/blob/96b2b739ebe601eabf415b0097c800f65dd551a4/prow/gerrit/client/client.go#L708-L727

Reply:
good catch, it's not required here, will update